pkgs/top-level/lua-packages.nix

```diff
@@ -270,6 +270,32 @@ let
     };
   };
 
+  luxio = buildLuaPackage rec {
+    name = "luxio-${version}";
+    version = "13";
+    src = fetchurl {
+      url = "https://git.gitano.org.uk/luxio.git/snapshot/luxio-luxio-13.tar.bz2";
+      sha256 = "1hvwslc25q7k82rxk461zr1a2041nxg7sn3sw3w0y5jxf0giz2pz";
+    };
+    nativeBuildInputs = [ which pkgconfig ];
+    postPatch = ''
+      patchShebangs .
+    '';
+    meta = {
+      platforms = stdenv.lib.platforms.unix;
+      license = stdenv.lib.licenses.mit;
+      description = "Lightweight UNIX I/O and POSIX binding for Lua";
```

Comment:
Do you want to maintain this package?

Reply:
Thanks for such a quick response! And also for fixing this up. I'll try and test subsequent PRs more thoroughly.

But yes I'm happy to be the maintainer for this and all related packages. :smiley:

Reply:
Ok then please add yourself to `lib/maintainers.nix` as well as to `meta.maintainers` field of this package.

Reply:
Done